include/LightningJSON/LightningJSON.hpp

```diff
@@ -136,11 +147,16 @@ namespace LightningJSON
 			return m_data[index];
 		}
 
-		std::string toString() const
+		operator std::string() const
```

Comment:
Can we keep both? I'm not opposed to a conversion operator, but I like having an explicit "ToString()" method.

I would, however, like the conversion operator to be marked explicit. I don't like implicit conversions unless there's a really strong argument for two types being used interchangeably and the conversion is very low-cost.